sw/device/silicon_creator/lib/base/static_critical_epmp.h

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
```

Comment:
Continuing from my comment above, I think we can remove this file, provide the declaration in `epmp.h`, weak definition in `epmp.c` and the actual definition in `static_critical_epmp.c`.

Reply:
I agree, I do that!